app/models/payment_receipt.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+class PaymentReceipt < ApplicationRecord
+  belongs_to :payment
+
+  scope :for_organization, lambda { |organization|
```

Comment:
What about adding the `organization_id` to this model?
It would removes the need for this complex query.